lms/djangoapps/grades/new/course_grade.py

```diff
@@ -3,10 +3,12 @@
 """
 
 from collections import defaultdict
+from logging import getLogger
+
 from django.conf import settings
 from django.core.exceptions import PermissionDenied
 from lazy import lazy
```

Comment:
Are you intentionally separate django imports from python imports?  If so, should `lazy` also be moved up?